.teamcity/settings.kts

```diff
@@ -71,6 +71,7 @@ project {
 		password("mc_teamcity_webhook", "credentialsJSON:7a711930-afd4-4058-b33f-39af8a0b7f91", display = ParameterDisplay.HIDDEN)
 		password("TRANSLATE_GH_APP_SECRET", "credentialsJSON:083cc9f7-4e9a-461f-b213-bc306baaeb28", display = ParameterDisplay.HIDDEN)
 		password("TRANSLATE_GH_APP_ID", "credentialsJSON:c03b1958-5ec3-4f4c-ab1c-ca1bf0e629f5", display = ParameterDisplay.HIDDEN)
+		password("SENTRY_AUTH_TOKEN", "credentialsJSON:6b927b24-85d6-4fad-a114-9acffc0b0dce", display = ParameterDisplay.HIDDEN)
```

Comment:
This is a personal auth token for now; we should update it to an organization integration auth token once we have permissions to do so

Reply:
Please let me know if/when you get something set up to do this.